browser/resources/settings/brave_routes.ts

```diff
@@ -50,7 +50,9 @@ export default function addBraveRoutes(r: Partial<SettingsRoutes>) {
   r.SOCIAL_BLOCKING.hasMigratedToPlugin = true
 
   r.EXTENSIONS = r.BASIC.createSection('/extensions', 'extensions')
+  r.EXTENSIONS.hasMigratedToPlugin = true
```

Comment:
@fallaciousreasoning for our edification, what does this do? I'm not familiar with this.

Reply:
Changed the architecture for how navigation works on the Settings page. This basically says to use the new way, not the old way.